utils/setStatus.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const statusCodes = require('http').STATUS_CODES
+    , setStatus = () => {
+        return function (status) {
+            /* eslint-disable no-invalid-this */
+            const that = this;
+            /* eslint-enable no-invalid-this */
+            let k;
+
+            if (status in statusCodes) {
+                that.statusCode = parseInt(status, 10);
+                that.statusMessage = statusCodes[status];
+                return;
+            }
+
+            for (k in statusCodes) {
```

Comment:
Would be more consistent within the code base to use `Object.keys(statusCodes).forEach((k) => { //your existing code here })` instead of a `for... in`